Sources/SwiftKuery/AuxiliaryTable.swift

```diff
@@ -16,25 +16,90 @@
 
 // MARK: AuxiliaryTable
 
-/// Subclasses of the `AuxiliaryTable` class are describing a table that is used in WITH clauses.
+/**
+ Subclasses of the `AuxiliaryTable` class are describing a `Table` that is used in WITH clauses.
+ ### Usage Example: ###
+ In this example, an `AuxiliaryTable` class is defined containing two columns.
+ A `ToDoTable` (as defined in `Table`), a `PersonTable` (as defined in `Column`) and a connection instances are initialized.
+ An instance of the `AuxiliaryTable` is then initialised from `Column` instances in "todotable".
+ This `AuxiliaryTable` is then used to create an SQL WITH query.
+ A description of the created query is then printed.
+ ```swift
+ class AuxTable: AuxiliaryTable {
+    let tableName = "aux_table"
+    let name = Column("name")
+    let finished = Column("finished")
+ }
+ 
+ let todotable = ToDoTable()          // ToDoTable() is a previously defined `Table` class
+ let persontable = PersonTable()      // PersonTable() is a previously defined `Table` class
+ let connection = createConnection()
+ 
+ let withTable = AuxTable(as: Select(todotable.toDo_completed.as("finished"), todotable.toDo_title.as("name"), from: todotable))
+ let withQuery = with(withTable, Select(withTable.finished, persontable.monthlyPay, from: persontable).join(withTable).on(persontable.name == withTable.name))
+ let stringQuery = connection.descriptionOf(query: withQuery)
```

Comment:
I don't know if it's because I used a connection pool to get my connection but I needed to change this line to
`let stringQuery = try connection?.descriptionOf(query: withQuery)`
to get this code to compile???